### PR TITLE
Tila 2834: add/rename testIDs

### PR DESCRIPTION
**Pull request by vincit-matu**

### CODE REVIEW

File: apps/ui/components/reservation/Step1.tsx [hunk 1]
```diff
@@ -166,6 +166,9 @@ const Step1 = ({
                     : typeof rawValue === "boolean"
                     ? t(`common:${String(rawValue)}`)
                     : rawValue;
+                  console.log(
+                    `${t(`reservationApplication:label.common.${key}`)} ${key}`
+                  );
```
Comment (joonatank) [inline apps/ui/components/reservation/Step1.tsx:171]: Necessary?

File: apps/ui/components/reservation/Step1.tsx [hunk 2]
```diff
@@ -218,6 +221,9 @@ const Step1 = ({
                 : typeof rawValue === "boolean"
                 ? t(`common:${String(rawValue)}`)
                 : rawValue;
+              console.log(
+                `${t(`reservationApplication:label.common.${key}`)} ${key}`
+              );
```
Comment (joonatank) [inline apps/ui/components/reservation/Step1.tsx:226]: Necessary?

File: apps/ui/components/reservation/EditStep1.tsx [hunk 3]
```diff
@@ -141,7 +143,7 @@ const EditStep1 = ({
       }
 
       if (key === "ageGroup") {
-        const { minimum, maximum } = reservation[key];
+        const { minimum, maximum } = reservation[key] ?? {};
         return `${minimum} - ${maximum}`;
```
Comment (joonatank) [inline apps/ui/components/reservation/EditStep1.tsx:147]: I'd default to something else here. like "1 - " if neither. minimum = 1, maximum = ""

**Review by joonatank — APPROVED** [review]
Check the few things otherwise LGTM

File: apps/ui/components/reservation/Step1.tsx (hunk 1 above)
Comment (vincit-matu) [reply apps/ui/components/reservation/Step1.tsx]: Nope, left there by mistake

**Review by joonatank — APPROVED** [review]

**Review by joonatank — APPROVED** [review]

**Pull request merged**